src/chrome/content/rules/National_Geographic.com.xml

```diff
@@ -0,0 +1,55 @@
+<!--
+
+	Redirect to HTTP:
+
+		- adventureblog
+		- channel
+		- help
+		- yourshot
+		- yourshotblog
+	
+	Mixed content:
+
+		- archive (Example: https://archive.nationalgeographic.com/?iid=133596#folio=CV1)
+		- intelligenttravel		
+		- news
+		- press
+		- video
```

Comment:
I can't find any mixed content, do you have an example?

(Note that we only care about **active** mixed content)